code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -592,16 +603,21 @@
 
 	//Citadel changes start here - Allows modules to use different icon files, and allows modules to specify a pixel offset
 	icon = (module.cyborg_icon_override ? module.cyborg_icon_override : initial(icon))
-
 	if(laser)
-		add_overlay("module.laser")//Is this even used??? - Yes modular_citadel/borg/inventory.dm
+		add_overlay("laser")//Is this even used??? - Yes borg/inventory.dm
 	if(disabler)
 		add_overlay("disabler")//ditto
 
 	if(sleeper_g && module.sleeper_overlay)
 		add_overlay("[module.sleeper_overlay]_g")
 	if(sleeper_r && module.sleeper_overlay)
 		add_overlay("[module.sleeper_overlay]_r")
+	if(resting)
```

Comment:
Might wanna add another check to make sure the borg actually has a resting icon

Reply:
There was, but the method used wasn't the same as the refactored code here, so it wasn't effective.